### ISSUE TITLE
OG Pi Zero support

### ISSUE THREAD

**Issue by shycoderX** [PaulJenkin/PS4RaspberryPi#32]
Would it be possible to add support for the non W Pi Zero?

**Comment by PaulJenkin** [comment]
Does it have a way to connect using Ethernet?

**Comment by shycoderX** [comment]
> Does it have a way to connect using Ethernet?

Yes using an USB to Ethernet dongle. You can also use a Wi-Fi dongle. But you need to connect a hub to the Pi then ofcourse so you can connect a usb port to the PlayStation too.

**Comment by EliasFogarty** [comment]
If you can SSH into the Pi try the command 'ifconfig' to find your adapter name (ex: wlan0)

**Comment by shycoderX** [comment]
> If you can SSH into the Pi try the command 'ifconfig' to find your adapter name (ex: wlan0)

AFAIK you need your PI connected to the internet to access SSH. I tried connecting to it via serial using an adapter (which works on RPi OS) but to no avail.

**Comment by shycoderX** [comment]
I think some drivers for USB Hubs and WiFi adapters are just missing which are available on RPi OS. As it detects my Hub and my WiFi adapter (as wlan0).
